Client/Frontend/SurveySurface/SurveySurfaceViewModel.swift

```diff
@@ -9,22 +9,18 @@ class SurveySurfaceViewModel {
     // MARK: - Properties
     weak var delegate: SurveySurfaceDelegate?
     var info: SurveySurfaceInfoProtocol
-    private var nimbus: FxNimbus
 
     // MARK: - Initialization
     init(with info: SurveySurfaceInfoProtocol,
-         delegate: SurveySurfaceDelegate,
-         and nimbus: FxNimbus = FxNimbus.shared
+         delegate: SurveySurfaceDelegate
     ) {
         self.info = info
         self.delegate = delegate
-        self.nimbus = nimbus
     }
 
     // MARK: - Functionality
     func didDisplayMessage() {
         delegate?.didDisplayMessage()
-        nimbus.features.messaging.recordExposure()
```

Comment:
no longer recording exposure?

Reply:
So, messaging exposure is recorded [here](https://github.com/mozilla-mobile/firefox-ios/blob/dfda1e230eb546bd23cf110a89dd4cb6e04fea3e/Client/Experiments/Messaging/GleanPlumbMessageManager.swift#L313). But, because of the bug, we weren't seeing it before, and adding it manually. But this fixes that, which makes our manual calls redundant. :)